### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DongrongLi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,15 @@ This repo tracks our double pass breadboards.  The goal is to realize stable, mo
...
 
-Assembled 2023-03 version:  
- <image img src="./2023-03-version/2023-03-assembled.jpg" width="600"></image>
+Assembled latest version (2024-06): 
+
+<img src="https://github.com/Jayich-Lab/double-pass-breadboard/assets/101778987/6a206f6d-f5c5-45f5-b2c1-621a959aeb96" height="600">
+
+## DPBB Schematic
+
+<img src="https://github.com/Jayich-Lab/double-pass-breadboard/assets/101778987/66780b9b-c1c0-4126-9dc8-c5642e20a401" width="500">
```
Comment (robertkwapisz) [inline README.md]: See https://jlab.zulipchat.com/#narrow/stream/305420-tech/topic/double.20pass.20breadboard/near/443515797 for Andrew's comments on the figures

**Comment by robertkwapisz** [comment]
The maximizing double pass efficiency section could be cleaned up to emphasize the important steps. Also, the second and third to-do items can easily be taken care of by the figures you add, so just do that and remove them from to-do.

**Comment by robertkwapisz** [comment]
I think the diagram lacks professionality. The spacing of elements is a bit off particularly the margins around the AOM. The labels should be placed consistent relative to the components. The line thickness all have nearly the same thickness and some components have border lines while others don't. Making this more consistent will go a long way in making this look more professional.

This from the moglabs manual is a nice example of a good looking diagram:

image

**Comment by robertkwapisz** [comment]
For the picture, as Andrew said before, a bit of an angle would help.

**Comment by robertkwapisz** [comment]
Looking at github commits it doesn't look like you made any changes to the text. I remember there being some typos and small parts that could be cleaned up. I don't have time to go through marking all of these right now.

**Comment by robertkwapisz** [comment]
> The maximizing double pass efficiency section could be cleaned up to emphasize the important steps. Also, the second and third to-do items can easily be taken care of by the figures you add, so just do that and remove them from to-do.

See this comment from my previous review

**Comment by DongrongLi** [comment]
> For the picture, as Andrew said before, a bit of an angle would help.

I thought the problem was with the extra cables and screws in the picture, also the fiber port was not visible in the last version. I cleared out the unwanted cables and screws around the DPBB, and this picture includes all the components as shown in the schematics. What seems to be the problem with this picture?

**Comment by DongrongLi** [comment]
A hard copy of the latest version is printed for Robert to review.

File: README.md [hunk 2]
```diff
@@ -5,60 +5,38 @@ This repo tracks our double pass breadboards.  The goal is to realize stable, mo
...
-
-* For more information on the Thorlabs collimator, see [manual](https://www.thorlabs.com/drawings/19effe7ca9179470-EC8BF155-96AC-6091-8774197AF94FBEB8/PAF2P-A10A-Manual.pdf).
-
-## Tips
-
-Here are tips for some specific problems that one may encounter while assembling a DPBB.
-
-* When IntraAction AOM shows low first pass efficiency/bandwidth, and you are quite certain that there is no issue with alignment, try flipping the AOM upside down and re-align.
-* Always use broadband dielectric plane mirrors for breadboard assembly. Using mirrors with curvature may significantly reduce the fiber coupling efficiency).
+* Couple the light into the output fiber using the two mirrors before the output port. For detailed instructions on how to fiber couple with [Thorlabs PAF2 Series Collimators](https://www.thorlabs.com/newgrouppage9.cfm?objectgroup_id=2940), see [Thorlabs manual](https://www.thorlabs.com/drawings/7d9ef555c8e29a3b-9ED63952-0AFC-37E4-8C5124F902B4BF9D/PAF2P-A10A-Manual.pdf) chapter 6.
```
Comment (robertkwapisz) [inline README.md]: the link to the manual does not work

File: README.md [hunk 3]
```diff
@@ -5,60 +5,38 @@ This repo tracks our double pass breadboards.  The goal is to realize stable, mo
...
 
+## DPBB Schematic
+<img src="https://github.com/Jayich-Lab/double-pass-breadboard/assets/101778987/4fecbb3c-94f0-4d26-af58-eb430a82806f" width = "600">
 
-# Assembly notes
 
-* Clean the machined parts before assembling.  
-* Center the laser light on the mirrors, waveplates, and PBS.  
-* Know the desired target center frequency and desired bandwidth.  
+# To-dos
```
Comment (robertkwapisz) [inline README.md]: Everything is under the to-dos section now?

**Pull request closed without merging**